code/server/routes/api.js

```diff
@@ -2,21 +2,13 @@ var _express = require('express');
 
 var router = _express.Router();
 var app = _express();
-var User = require('../models/User.js');
-var UserMatches = require('../models/UserMatches.js');
+var UserDAO = require('../data_access/UserDataAccess.js');
+var UserMatchDAO = require('../data_access/UserMatchDataAccess.js');
 var authenticator = require("../mixins/authenticator.js");
 
-var injectUser = function(user) {
-	if (user) {
-		User = user;
-	}
-};
-
-var injectLikes = function(likes) {
-	if (likes) {
-		UserMatches = likes;
-	}
-};
+// Initialize models to use DB
+UserDAO.init('db');
+UserMatchDAO.init('db');
```

Comment:
Yeah, I'm not sure of a nicer way to do it either... but I will be adding a function to inject the stub as mentioned above.